crates/iota-replay/src/tests.rs

```diff
@@ -16,11 +16,10 @@ use crate::{
 
 /// Keep searching for non-system TXs in the checkppints for this long
 /// Very unlikely to take this long, but we want to be sure we find one
-const NUM_CHECKPOINTS_TO_ATTEMPT: usize = 1_000;
+const NUM_CHECKPOINTS_TO_ATTEMPT: usize = 10_000;
```

Comment:
Is this increase required for it to pass?

Reply:
We currently have not enough user transactions in the testnet 😅